tsconfig.json

```diff
@@ -2,17 +2,19 @@
   "compilerOptions": {
     "outDir": "./dist",
     "sourceMap": true,
-    // "strictNullChecks": true,
+    "strictNullChecks": true,
+    "allowJs": true,
```

Comment:
Why did we allow this?

Reply:
this came up in our last meeting on polykey, it is recommended when using typescript. It requires you to explicitly specify an `undefined` or `null` type instead of existing in the domain of every type. This is advantageous when want to force a required parameter not to be null like `path` in `readdirSync`

Reply:
Test not having this option and see if it works.

Reply:
webpack, tsc and jest can all run without this option so I've removed it.